validator/test/fixtures/constraints/forbid_world_readable_storage_bucket/data.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPStorageWorldReadableConstraintV1
+metadata:
+  name: require_storage_iam
+  annotations:
+    # Example of tying a constraint to a CIS benchmark
+    benchmark: CIS11_5.03
+spec:
+  constraintVersion: 0.1.0
```

Comment:
let's delete constraintVersion here; it's not used (we encode the version as part of the constraint template name now)

Reply:
Done.